### ISSUE TITLE
No image in the title cell in the "Photo to Anime with PaddleGAN and OpenVINO" notebook

### ISSUE THREAD

**Issue by artyomtugaryov** [openvinotoolkit/openvino_notebooks#152]
Hello,
I tried to run notebook and found that the image with dog is not showing in the first cell of the notebook:
![image](https://user-images.githubusercontent.com/23581254/124613058-86de6e80-de7b-11eb-9f49-c9952cafd5c2.png)

But I can open the link with this image.
I found in the console of my browser the error: 
![image](https://user-images.githubusercontent.com/23581254/124613303-c1e0a200-de7b-11eb-9970-8bfa294be461.png)

I run the notebook inside a Docker container

**Comment by helena-intel** [comment]
Thanks for reporting this @artyomtugaryov ! Can you share which Docker image you used? I have tried the notebook in the Docker image built with the Dockerfile in this repository and the image shows there.

**Comment by artyomtugaryov** [comment]
Actually, I used the OpenVINO DL Workbench Docker 2021.4 release - openvino/workbench:2021.4

**Comment by helena-intel** [comment]
Thanks! It seems that that Docker image doesn't support loading images from external sources. I will look into the best way to address this. Note that DL Workbench is not yet a supported environment for the OpenVINO notebooks though. Starting from the next OpenVINO release (2022.1) it will be.

I tried running the PaddleGAN notebook in DL Workbench. It did not immediately work, because of a requirements mismatch, but after I added `--user` to the `pip install` command for the paddlegan requirements, and adding `!pip install gast==0.3.3 requests==2.25.1 --user` it worked. I don't recommend this on a production system though.
